internal/store/postgres/model/grant.go

```diff
@@ -90,6 +93,26 @@ func (m *Grant) FromDomain(g domain.Grant) error {
 		m.RevokedAt = *g.RevokedAt
 	}
 
+	if g.RestoreReason != "" {
+		m.RestoreReason = sql.NullString{
+			String: g.RestoreReason,
+			Valid:  true,
+		}
+	}
```

Comment:
```suggestion
	m.RestoreReason = sql.NullString{
	    String: g.RestoreReason,
	    Valid:  g.RestoreReason != "",
         }
```
this would remove if block and save 2 lines. we could also consider creating helper to make it one line.

```suggestion
	m.RestoreReason = newNullString(g.RestoreReason)
	m.RestoreAt = newNullTime(g.RestoredAt)
```

note: can ignore this to follow the existing pattern.